tuned/ppd/controller.py

```diff
@@ -0,0 +1,220 @@
+from tuned import exports, logs
+from tuned.utils.commands import commands
+from enum import StrEnum
+import time
+import dbus
+import os
+
+log = logs.get()
+
+DRIVER = "TuneD"
+NO_TURBO_PATH = "/sys/devices/system/cpu/intel_pstate/no_turbo"
+LAP_MODE_PATH = "/sys/bus/platform/devices/thinkpad_acpi/dytc_lapmode"
+
+
+class PPDProfile(StrEnum):
```

Comment:
Sure, I can do that. What exactly do you propose that the `hold` section does? Currently, the daemon allows holding only `power-saver` and `performance` profiles, similar to PPD.

Reply:
> Sure, I can do that. What exactly do you propose that the `hold` section does? Currently, the daemon allows holding only `power-saver` and `performance` profiles, similar to PPD.

I wanted to map min performance and max performance hold profiles to PPD profile names, but because the mapping to TuneD profiles would be already under the `[profiles]` section, this is probably redundant abstraction. Let's don't over-engineer it and say `power-saver` and `performance` profiles mapping have to be always defined in the `[profiles]` section.

Otherwise I think it shouldn't be hard change, just one more INI parser and I think it's worth the benefit of greater flexibility.

Reply:
Right, I agree, let's keep it simple.